app/controllers/needs_controller.rb

```diff
@@ -160,6 +160,9 @@ def filtered_params
       met_when: [],
       organisation_ids: [],
       justifications: [],
+      status: [
+        :description,
```

Comment:
Looking at [Maslow](https://github.com/alphagov/maslow/blob/master/app/models/need_status.rb#L23), we need to add a few more attributes for every status change to go through.